plugins/infinispan90/src/main/java/org/radargun/service/Infinispan70CacheInfo.java

```diff
@@ -0,0 +1,47 @@
+package org.radargun.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.infinispan.AdvancedCache;
+import org.infinispan.container.DataContainer;
+import org.infinispan.distribution.ch.ConsistentHash;
+import org.radargun.traits.CacheInformation;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ * @author Alan Field &lt;afield@redhat.com&gt;
+ */
+public abstract class Infinispan70CacheInfo extends Infinispan53CacheInfo {
```

Comment:
It seems kind of confusing to use the `Infinispan70CacheInfo` name in the `infinispan90` plugin code?